config/workers.coffee

```diff
@@ -195,6 +330,31 @@ module.exports = (KONFIG, options, credentials) ->
             internalOnly: yes
           }
         ]
+      kubernetes        :
+        image           : 'koding/base'
+        command         : " [ \"./run\", \"exec\", \"#{nodeProgram}\", \"#{KONFIG.projectRoot}/workers/notification\", \"-p\", \"4560\" ] "
+        mounts          : [ 'koding-working-tree', 'root-kite-volume' ]
+        ports						:
+          containerPort : '4560'
+          hostPort			: '4560'
```

Comment:
Much space.

Reply:
My drunk coder detector is detecting a drunk coder :trollface: